contracts/ColonyTask.sol

```diff
@@ -332,10 +338,11 @@ contract ColonyTask is ColonyStorage {
   function setTaskDomain(uint256 _id, uint256 _domainId) public
   stoppable
   taskExists(_id)
-  domainExists(_domainId)
   taskNotComplete(_id)
   self()
   {
+    require(domainExists(_domainId), "colony-domain-does-not-exist");
```

Comment:
🤔 and you removed the modifier because?

Reply:
It was redundant in all the other cases because `authDomain` checks if the domain exists. Since there was only one existing `domainExists` check which wasn't preceded by an `authDomain` check (in this function), I figured make `domainExists` into a function, use it in `authDomain`, and then throw it into a `require` this one time.